pkg/predict/input.go

```diff
@@ -25,15 +28,16 @@ func NewInputs(keyVals map[string]string) Inputs {
 		val := val
 		if strings.HasPrefix(val, "@") {
 			val = val[1:]
-			expandedVal, err := homedir.Expand(val)
-			if err != nil {
-				// FIXME: handle this better?
-				console.Warnf("Error expanding homedir: %s", err)
+			input[key] = Input{File: &val}
+		} else if strings.HasPrefix(val, "[") && strings.HasSuffix(val, "]") {
+			// Handle array of strings
+			var arr []interface{}
+			if err := json.Unmarshal([]byte(val), &arr); err != nil {
```

Comment:
Is JSON unmarshalling right here? That means the strings have to be quoted, which is inconsistent with the other types.

For example, I can do

```
cog predict hello-world -i text=world
```

or

```
cog predict resnet -i image=@input.jpg
```

but with this implementation, the following will throw an error

```
cog predict mymodel -i names=[foo,bar]
```

I think there are two possibilities here. Either:

1. do the work to allow individual strings to be either quoted or unquoted, so that they're consistent with the rest of the interface, or 
2. change approach to do arrays by just repeating inputs, e.g. `cog predict ... -i names=one -i names=two`

Reply:
I chose repeated inputs. Good idea. Thanks.